i18n/nl/cosmic_settings.ftl

```diff
@@ -0,0 +1,807 @@
+        [one] dag
+       *[other] dagen
+  }
+  .less-than-minute = In minder dan één minuut
+  .and = en
+  .remaining-time = { $time } { $action ->
+        [full] opgeladen
+       *[other] leeg
+   }
+
+connected-devices = Verbonden apparaten
+  .unknown = Onbekend apparaat
+
+power-mode = Energieverbruik
+    .battery = Energiebesparing
+    .battery-desc = Verminderd stroomverbruik en geruisloze prestaties.
+    .balanced = Gebalanceerd
+    .balanced-desc = Gemiddeld stroomverbruik en stille prestaties.
+    .performance = Hoge Prestatie
+    .performance-desc = Hoge prestatie en verhoogd stroomgebruik.
+    .no-backend = Backend niet gevonden. Installeer system76-power of power-profiles-daemon.
+
+power-saving = Enegriebesparingsopties
+    .turn-off-screen-after = Beeldscherm uitschakelen na
+    .auto-suspend = Automatische slaapstand
+    .auto-suspend-ac = Automatische slaapstand wanneer op het lichtnet aangesloten
+    .auto-suspend-battery = Automatische slaapstand wanneer op batterijstroom
+
+## Input
+
+acceleration-desc = De trackinggevoeligheid wordt automatisch aangepast op basis van de snelheid.
+
+disable-while-typing = Tijdens het typen uitschakelen.
+
+input-devices = Invoerapparaten
+    .desc = Invoerapparaten
+
+primary-button = Primaire knop
+    .desc = Bepaalt de rangschikking van de fysieke toetsen.
+    .left = Links
+    .right = Rechts
+
+scrolling = Scrollen
+    .two-finger = Met twee vingers scrollen
+    .edge = Scroll met een vinger langs de rand
+    .speed = Scrollsnelheid
+    .natural = Natuurlijk scrollen
+    .natural-desc = Scroll de inhoud, niet het weergavevenster.
+
+## Input: Keyboard
+
+slow = Langzaam
+fast = Snel
+short = Kort
+long = Lang
+keyboard = Toetsenbord
+    .desc = Invoermethodes, omschakelen, invoer van speciale tekens en sneltoetsen.
+
+keyboard-sources = Invoermethodes
+    .desc = Invoermethodes kunnen worden gewisseld met de sneltoetscombinatie Super+Spatie. Dit kan aangepast worden in de instellingen voor sneltoetsen.
+    .move-up = Omhoog
+    .move-down = Naar beneden
+    .settings = Instellingen
+    .view-layout = Toetsenbordindeling weergeven
+    .remove = Verwijderen
+    .add = Invoermethode toevoegen
+
+keyboard-special-char = Invoer speciale tekens
+    .alternate = Toets voor speciale tekens
+    .compose = Compose-toets (voor speciale tekens)
+    .caps = Caps Lock-toets
+
+keyboard-typing-assist = Typen
+    .repeat-rate = Snelheid van herhalen
+    .repeat-delay = Herhalingsvertraging
+
+added = Toegevoegd
+type-to-search = Typ om te zoeken…
+show-extended-input-sources = Toon uitgebreide invoermethodes
+
+## Input: Keyboard: Shortcuts
+
+keyboard-shortcuts = Sneltoetsen
+    .desc = Sneltoetsen bekijken en aanpassen
+
+add-keybinding = Sneltoets toevoegen
+cancel = Annuleren
+command = Commando
+custom = Aangepast
+debug = Fouten opsporen
+disabled = Uitgeschakeld
+migrate-workspace-prev = Werkblad naar vorige uitvoer verplaatsen
```

Comment:
line 540 to 580: 
What's the difference between an output and a monitor,
in dutch: "uitvoer" or "uitvoerbron" and "beeldscherm" or just "scherm"